### PR TITLE
drop PHP 5.4 code check

### PR DESCRIPTION
**Pull request by samsonasik**

### CODE REVIEW

File: src/CallbackHandler.php [hunk 1]
```diff
@@ -84,15 +78,9 @@ public function getCallback()
...
-
-        // Minor performance tweak, if the callback gets called more than once
-        if (!isset(static::$isPhp54)) {
-            static::$isPhp54 = version_compare(PHP_VERSION, '5.4.0rc1', '>=');
-        }
-
         $argCount = count($args);
 
-        if (static::$isPhp54 && is_string($callback)) {
+        if (is_string($callback)) {
```
Comment (manuakasam) [inline src/CallbackHandler.php]: This should be wrong as the operator was `&&`, thus the whole condition shouldn't be required without it being 5.4

**Comment by Ocramius** [comment]
Merged, thanks!

`master`: zendframework/zend-stdlib@c9fd305f50fee4427aa200a9959ad935b38a8b06
`develop`: zendframework/zend-stdlib@af2328eb12fe33ca39f7f3f07885e8aceb68d79f

**Comment by cebe** [comment]
Did you really just bumped the PHP version requirement from 5.3 to 5.5 on a minor version release here? Or am I overlooking something?

**Comment by Ocramius** [comment]
@cebe yes, and it has been discussed multiple times: not a BC break

**Comment by cebe** [comment]
Can you point me to a discussion that came to that conclusion? Imo when I am having a System on PHP 5.4 and I update my ZF component from 2.5.1 to 2.5.2 and it stops working, that is a BC break.

**Comment by akrabat** [comment]
From http://framework.zend.com/blog/zend-framework-2-5-0-released.html:

> Finally, we have a significant bump in our minimum supported PHP version starting with 2.5.0: we now require PHP 5.5+. PHP 5.3 is no longer receiving even security updates at this time, and PHP 5.4 support ends on 14 September 2015 — just over 3 months from now. We encourage our users on versions prior to 5.5 to migrate immediately; the migration from 5.4 to 5.5 or 5.6 is trivial, and you'll get substantial performance benefits in addition to new features.

**Comment by akrabat** [comment]
If you must continue to deploy on PHP 5.4 after 14th September 2015, then use ZF 2.4.

**Comment by cebe** [comment]
okay, so it seems you just forgot to update the requirement in composer.json in 2.5.0. Thanks for the link and info! Will use 2.4 for now.

**Comment by Maks3w** [comment]
Yep, There was a confusion when 2.5.0 was published. Sorry for the inconvenience.

**Comment by Maks3w** [comment]
PD: PHP 5.4 will reach EOL and no more security patches will receive.

**Pull request merged**